command/tls_ca_create.go

```diff
@@ -0,0 +1,162 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/nomad/helper/flags"
+	"github.com/hashicorp/nomad/helper/tlsutil"
+	"github.com/hashicorp/nomad/lib/file"
+	"github.com/posener/complete"
+)
+
+type TLSCACreateCommand struct {
+	Meta
+
+	// days is the number of days the CA will be valid for
+	days int
+
+	// constraint provides a spoecific name constraint to the CA which will then
+	// reject any domains other than this.
+	constraint bool
+
+	// domain is used to provide a custom domain for the CA
+	domain string
+
+	// commonName is used to set a common name for the CA
+	commonName string
+
+	// additioanlNameConstraint provides a list of name constraints to the CA which will then
+	// reject any domains other than these.
+	additionalNameConstraint flags.StringFlag
+}
+
+func (c *TLSCACreateCommand) Help() string {
+	helpText := `
+Usage: nomad tls ca create [options]
+
+Create a new Certificate Authority.
+
+Here are some simple examples, more details can be found below or in 
+the documentation.
+
+Create a new Nomad CA
+
+  $ nomad tls ca create
+  ==> CA Certificate saved to: nomad-agent-ca.pem
+  ==> CA Certificate key saved to: nomad-agent-ca-key.pem
+
+  Create a new Nomad CA with a Custiom Domain
```

Comment:
```suggestion
  Create a new Nomad Certificate Authority
```

But this is also repeated above